.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,70 @@
+---
+name: Bug report
+about: Submit the behavior you consider invalid
+
+---
+
+<!--
+If you have all reproduction steps with a complete sample app, please share as many details as possible in the sections below.
+
+Make sure that you tried using the latest TestCafe version (https://github.com/DevExpress/testcafe/releases), where this behavior might have been already addressed.
+
+Before submitting an issue, please check CONTRIBUTING.md and existing issues in this repository (https://github.com/DevExpress/testcafe/issues) in case a similar issue exists or was already addressed. This may save your time (and ours).
+-->
+
+### What is your Test Scenario?
+<!-- A clear and concise description of what you'd like to test. -->
+ 
+### What is the Current behavior?
+<!-- A clear and concise description of the behavior you see and consider invalid. -->
+ 
+### What is the Expected behavior?
+<!-- A clear and concise description of what you expected to happen. -->
```

Comment:
Aren't we being too aggressive with this `clear and concise` everywhere? Can we be more friendly instead?

Reply:
Yes, probably. This is the default text when adding a new issue template. Revering back to our own version.